src/tactic/ui/tools/ingest_wdg.py

```diff
@@ -1602,26 +1604,24 @@ def execute(my):
             else:
                 path = filename
 
-            # extract keywords from filename
-            file_keywords = Common.extract_keywords_from_path(path)
+            # Extracting keywords from filename only because all Assets will 
+            # have the keywords [project_name]/[search_type name]/[Ingest]
+            file_keywords = Common.extract_keywords_from_path(filename)
```

Comment:
# this is debatable.. Should we extract keywords from the full path if relative_dir is not used for the IngestUploadWdg?